cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -1129,7 +1129,9 @@ var CMS = window.CMS || {};
                 // collapsable function and save states
                 if (el.hasClass('cms-dragitem-expanded')) {
                     settings.states.splice($.inArray(id, settings.states), 1);
-                    el.removeClass('cms-dragitem-expanded').parent().find('> .cms-draggables').addClass('cms-hidden');
+                    el.removeClass('cms-dragitem-expanded').parent()
+                        .find('.cms-collapsable-container').eq(0).addClass('cms-hidden');
```

Comment:
` .find('.cms-collapsable-container').eq(0)` → `.find('> .cms-collapsable-container')`

technically they are not the same